### ISSUE TITLE
Ability to rotate rulers

### ISSUE THREAD

**Issue by PG979** [pascalpp/FreeRuler#113]
It would be nice to be able to rotate the rulers on the screen an arbitrary amount, to be able to measure distances that are neither horizontal or vertical on the screen.

This would be even more useful if combined with the (separately requested) ability to scale the rulers.

Thanks for the nice tool!

**Comment by julllleee** [comment]
Yes I agree, love the tool however some way to rotate would be a killer feature.

Thanks for your hard work! 🙂

**Comment by SammmyAhmad** [comment]
Yes, both features requested - would be very helpful
